resources/CMakeLists.txt

```
@@ -36,6 +36,11 @@ endif()
 
 # Server landingpage webapp
 if (WITH_SERVER_LANDINGPAGE_WEBAPP)
+  find_program(YARN yarn)
+  if(NOT YARN)
+    message(FATAL_ERROR "Yarn not found, required for server landingpage webapp!")
+  endif()
+
```

```suggestion
  find_program(YARN yarn REQUIRED)

```

I think that should do

---

Now below, use `    COMMAND cd ${LANDINGPAGE_OUTPUT_PATH} && ${YARN} install --frozen-lockfile && ${YARN} build
`

---

Thanks and done.